packages/angular-table/src/flexrender.directive.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Directive,
+  Input,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core'
+
+@Directive({
+  selector: '[flexRender]',
+  standalone: true,
+})
+export class FlexRenderDirective implements OnInit {
+  // private vcr = inject(ViewContainerRef)
+  // private templateRef = inject(TemplateRef)
+
+  private _flexRender: any
+
+  /** properties to render */
+  private _flexRenderProps: any
+
+  @Input({ required: true })
+  set flexRender(render: any) {
```

Comment:
Same here, let add constraints on the input type.

Reply:
is this looks good or any suggestions

type FlexRenderType = string | null | (() => any) | ((props: any) => any)

Reply:
tried changing the type getting the following error 
Type 'ColumnDefTemplate<HeaderContext<Person, unknown>> | undefined' is not assignable to type 'FlexRenderType'.